### ISSUE TITLE
Doesn't accept new admin account password after changing it[BUG]

### ISSUE THREAD

**Issue by RaananZemer** [grahampugh/erase-install#519]
Doesn't accept new admin user password after changing it saying 
Error - Password is invalid
using older passwords does no work as well

**Comment by RaananZemer** [comment]
I found a way to bypass this issue
I create another Admin account and use it to do the process and it works

**Comment by grahampugh** [comment]
Please supply logs.

**Comment by RaananZemer** [comment]
It only happened to me twice and all macbooks I erased after that did not have this issue
I have no more macbooks to erase right now but if it will happen again I will upload the logs

**Comment by RaananZemer** [comment]
[erase-install.log](https://github.com/user-attachments/files/16438243/erase-install.log)
Hi, this issue happened again
This is the log

**Comment by grahampugh** [comment]
Can you explain to me exactly what you did to get this issue? How exactly and when did you change the admin password? 

Can you reproduce this when using startosinstall directly?

**Comment by RaananZemer** [comment]
I got this when I changed user password password in 2 ways
The first is when I reset it through recovery using a recovery key
the second change type was when I pushed for a password change through a JAMF policy - I can log in using that password so it is correct

Running 'startosinstall' seems to be working
after entering the password has started the process and it is now preparing to run

**Comment by RaananZemer** [comment]
Hi,
Most of our fleet has Intel silicon Macbooks so you might be correct. But, this didn't happen for every Mac I formatted and did the same process.
In any case, as this is an exception and not the rule I have a workaround. 
I create another admin account on the device and use it to format.

**Comment by grahampugh** [comment]
Don't forget this account does not need to be an administrator. But it does need to have a Secure Token. I wonder if your LAPS rotation is not updating the Secure Token? I believe this would be the case if the Jamf LAPS account was created via a PreStage Enrollment, but not if it was created as the Management Account.

**Comment by RaananZemer** [comment]
Making an admin account makes it easier for me but you are right, it doesn't necessary
Currently, I don't have a LAPS configured for our pre-enrollment admin account

**Comment by GabeShack** [comment]
> Don't forget this account does not need to be an administrator. But it does need to have a Secure Token. I wonder if your LAPS rotation is not updating the Secure Token? I believe this would be the case if the Jamf LAPS account was created via a PreStage Enrollment, but not if it was created as the Management Account.

Yes I believe this was the case for us.  Jamf didnt acknowledge the issue, but once I made a script to auto change the laps password back to the originally created password, everything worked as intended.  (funny that also while the laps password rotations were happening, other users were also not being granted secure tokens or volume ownership).
